content/dart/concepts/list/list.md

```diff
@@ -0,0 +1,155 @@
+---
+Title: 'List'
+Description: 'A list provides a way to organize and store a collection of items.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Lists'
+  - 'Arrays'
+  - 'Objects'
+  - 'Data Structures'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+A **list** provides a way to organize and store a collection of items. Simply, lists act as containers that can hold different pieces of data, such as strings, integers, boolean values, or even other lists. Lists are a great way to organize related data in an easy-to-manipulate format.
+<!-- I think will be worth linking to other docs pages like 'strings' when they have been completed/submitted. -->
+
+## Syntax
+
+In Dart, there are two ways to create a list, there is the more prominently used method of `var` and the lesser used `List` class, both use square brackets (`[]`) and commas (`,`). The square brackets are used to define the data, whilst commas are used to separate the individual items.
+
+First, an example of utilizing `var` to create a list:
+
+```dart
+// Create a list of strings
+var favouriteFruits = ['Apple', 'Banana', 'Orange'];
+
+// Create a list of integers
+var oddNumbers = [1, 3, 5, 7, 9];
+
+// Create a list of lists (nested list)
+var nestedList = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+];
+```
+
+Then, an example of utilizing `List` to create a list:
+```dart
+// Create a list of strings
+List favouriteFruits = ['Apple', 'Banana', 'Orange'];
+
+// Create a list of integers
+List oddNumbers = [1, 3, 5, 7, 9];
+
+// Create a list of lists (nested list)
+List nestedList = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+];
+```
+
+>**Note:** As `var` is the predominantly used method to create lists in Dart, the rest of the examples will use this moving forward.
+
+In the above examples, there are three separate lists:
+- A list of strings
+- A list of integers
+- A nested list
```

Comment:
You can add this before the example b=code block. It would be better for the users to understand the code block.